app/views/analytics/districts/show.html.erb

```diff
@@ -1,130 +0,0 @@
-<%= render "shared/old_reports_banner" %>
-
-<div class="dashboard">
-  <nav class="breadcrumb">
-    <%= link_to 'All districts', root_path %>
-    <i class="fas fa-chevron-right"></i>
-
-    <%= @organization_district.district_name %>
-  </nav>
-
-  <%= render "period_dropdown" %>
-
-  <h1><%= @organization_district.district_name %></h1>
-</div>
-
-<%= render 'shared/cohort_charts', cohort_analytics: @cohort_analytics, quarter: current_quarter, year: current_year %>
-<% if current_admin.permissions_v2_enabled? %>
-  <%= render 'follow_ups_table',
```

Comment:
There are some partials like this sitting in this directory that can now be moved to the `reports/region` area, since those are the only reports that use em now.

Reply:
Yup, removed those.